packages/opds-web-client/src/utils/file.ts

```diff
@@ -44,6 +44,10 @@ export const typeMap: TypeMap = {
   "text/html;profile=http://librarysimplified.org/terms/profiles/streaming-media": {
     extension: "",
     name: "streaming-media"
+  },
+  "application/audiobook+json": {
+    extension: ".audiobook",
```

Comment:
@leonardr is this right?

Reply:
I think `.audiobook-manifest` is more accurate. That's what we use in core/model/constants.py and that's what the example has in https://github.com/readium/webpub-manifest/blob/master/extensions/audiobook.md.